modules/install-osp-deploy-dualstack.adoc

```diff
@@ -138,4 +138,30 @@ platform:
 <4> Specify the dual-stack network details that are used by all the nodes across the cluster.
 <5> The CIDR of any subnet specified in this field must match the CIDRs listed on `networks.machineNetwork`.
 <6> You can specify a value for either `name` or `id`, or both.
-<7> Specifying the `network` under the `ControlPlanePort` field is optional.
\ No newline at end of file
+<7> Specifying the `network` under the `ControlPlanePort` field is optional.
+
+[NOTE]
```

Comment:
This is an optional suggestion, but you might consider making this note a separate troubleshooting module. It seems pretty important, but users might miss it as a note within a procedure.